### PR TITLE
[office-js][office-js-preview] (Outlook) Clarify appointment location types

### PR DESCRIPTION
**Pull request by samantharamon**
- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/OfficeDev/office-js/issues/2944
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## johnny-five ([<kbd>unpkg</kbd>](https://unpkg.com/browse/johnny-five@latest/))
was missing the following properties:
1. Barometer
1. Color
1. Expander
1. GPS
1. Gripper

<details>
<summary>as well as these 22 other properties...</summary>
<p>Keypad, LedControl, Light, Orientation, Repl, Wii, Multi, Analog, Digital, Temperature, Luxmeter, Magnetometer, Boards, Buttons, Leds, Pins, Relays, Sensors, Servos, Switches, Touchpad, Nunchuk</p>
</details>

## selenium-webdriver ([<kbd>unpkg</kbd>](https://unpkg.com/browse/selenium-webdriver@latest/))
was missing the following properties:
1. ThenableWebDriver
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 48e10b1a159ade68d6a3523711fa8d815c14e4f8
</p>

File: types/office-js-preview/index.d.ts [hunk 1]
```diff
@@ -8862,11 +8862,15 @@ declare namespace Office {
          */
         enum LocationType {
             /**
-             * A custom location.
+             * A custom location. Custom locations don't have an SMTP address.
+             * 
+             * **Note**: {@link https://support.microsoft.com/office/create-a-contact-group-or-distribution-list-in-outlook-for-pc-88ff6c60-0a1d-4b54-8c9d-9e1a71bc3023 | Personal contact groups}
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts:8867]:
```suggestion
             * **Note**: {@link https://support.microsoft.com/office/88ff6c60-0a1d-4b54-8c9d-9e1a71bc3023 | Personal contact groups}
```

File: types/office-js-preview/index.d.ts [hunk 2]
```diff
@@ -13144,6 +13148,9 @@ declare namespace Office {
         addAsync(locationIdentifiers: LocationIdentifier[], callback?: (asyncResult: Office.AsyncResult<void>) => void): void;
         /**
          * Gets the set of locations associated with the appointment.
+         * 
+         * **Note**: {@link https://support.microsoft.com/office/create-a-contact-group-or-distribution-list-in-outlook-for-pc-88ff6c60-0a1d-4b54-8c9d-9e1a71bc3023 | Personal contact groups}
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts:13152]:
```suggestion
         * **Note**: {@link https://support.microsoft.com/office/88ff6c60-0a1d-4b54-8c9d-9e1a71bc3023 | Personal contact groups}
```

File: types/office-js-preview/index.d.ts [hunk 3]
```diff
@@ -13161,6 +13168,9 @@ declare namespace Office {
         /**
          * Gets the set of locations associated with the appointment.
          *
+         * **Note**: {@link https://support.microsoft.com/office/create-a-contact-group-or-distribution-list-in-outlook-for-pc-88ff6c60-0a1d-4b54-8c9d-9e1a71bc3023 | Personal contact groups}
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js-preview/index.d.ts:13171]:
```suggestion
         * **Note**: {@link https://support.microsoft.com/office/88ff6c60-0a1d-4b54-8c9d-9e1a71bc3023 | Personal contact groups}
```

File: types/office-js/index.d.ts [hunk 4]
```diff
@@ -8735,11 +8735,15 @@ declare namespace Office {
          */
         enum LocationType {
             /**
-             * A custom location.
+             * A custom location. Custom locations don't have an SMTP address.
+             * 
+             * **Note**: {@link https://support.microsoft.com/office/create-a-contact-group-or-distribution-list-in-outlook-for-pc-88ff6c60-0a1d-4b54-8c9d-9e1a71bc3023 | Personal contact groups}
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js/index.d.ts:8740]:
```suggestion
             * **Note**: {@link https://support.microsoft.com/office/88ff6c60-0a1d-4b54-8c9d-9e1a71bc3023 | Personal contact groups}
```

File: types/office-js/index.d.ts [hunk 5]
```diff
@@ -12882,6 +12886,9 @@ declare namespace Office {
         /**
          * Gets the set of locations associated with the appointment.
          *
+         * **Note**: {@link https://support.microsoft.com/office/create-a-contact-group-or-distribution-list-in-outlook-for-pc-88ff6c60-0a1d-4b54-8c9d-9e1a71bc3023 | Personal contact groups}
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js/index.d.ts:12889]:
```suggestion
         * **Note**: {@link https://support.microsoft.com/office/88ff6c60-0a1d-4b54-8c9d-9e1a71bc3023 | Personal contact groups}
```

File: types/office-js/index.d.ts [hunk 6]
```diff
@@ -12898,6 +12905,9 @@ declare namespace Office {
...
```
Comment (ElizabethSamuel-MSFT) [inline types/office-js/index.d.ts:12908]:
```suggestion
         * **Note**: {@link https://support.microsoft.com/office/88ff6c60-0a1d-4b54-8c9d-9e1a71bc3023 | Personal contact groups}
```

File: types/office-js-preview/index.d.ts (hunk 1 above)
Comment (samantharamon) [reply types/office-js-preview/index.d.ts:8867]: Thanks for catching this!

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Pull request merged**